### ISSUE TITLE
Visualization problem

### ISSUE THREAD

**Issue by harunijaz** [zhijieW-94/SAGNet#10]
I found your work very interesting. however the given link to Anaconda environment  for cloning doesn't exist while in the meantime I couldn't find VTKv5.10.1. I've set VTKv8.1.0. the program runs fine but generate blank sections upon visualizing as shown in screenshot. 
Is there any possible solution to solve this visualization problem or am I missing something?

![image](https://github.com/zhijieW94/SAGNet/assets/26218942/c56bd33b-71b3-4372-9d3d-6d701a9e0716)

**Comment by harunijaz** [comment]
I've resolved the problem, and I'd like to share the solution to save time for others.

I've set up two separate Conda environments: one for generating test/train data and another for visualization, with Python versions 2.7 and 3.6, respectively. Here are the key versions I used, that are different from the project's home page:

   1. TensorFlow-GPU v1.3.0 (Conda environment Python versions 2.7)
   2.  VTK v8.1.2 (Conda environment Python versions 3.6)
   3.  PyQt5 v5.15.6 (Conda environment Python versions 3.6)

I've made necessary code changes to accommodate the VTK upgrade.
-----------------------------------------
FILE NAME: Visualization/gui_viewer.py
(line 65)  gfilter.SetInput(ugrid) >>>  gfilter.SetInputData(ugrid) 
(line 89) self.glyph3D_list[i].SetSource(voxel.GetOutput())   >>> self.glyph3D_list[i].SetSourceConnection(voxel.GetOutputPort())
(line 90) same as line 65 above
(line 96) mapper.SetInput(self.glyph3D_list[i].GetOutput())   >>>  mapper.SetInputConnection(self.glyph3D_list[i].GetOutputPort())
line(285 & 294)  window_to_image_filter.SetInputData(self.GetRenderWindow())  >>> 
        window_to_image_filter.SetInput(self.GetRenderWindow())
line(288 & 297) unicode >>> str 

I hope it would work. cheers!!!
